jib-core/src/main/java/com/google/cloud/tools/jib/ncache/DefaultCacheStorageWriter.java

```diff
@@ -108,8 +108,11 @@ CacheEntry write(CacheWrite cacheWrite) throws IOException {
     // Creates the layers directory if it doesn't exist.
     Files.createDirectories(defaultCacheStorageFiles.getLayersDirectory());
 
-    // Creates the temporary directory.
-    try (TemporaryDirectory temporaryDirectory = new TemporaryDirectory()) {
+    // Creates the temporary directory. The temporary directory must be in the same FileStore as the
```

Comment:
`{@link FileStore}` and `{@code Files.move}`?

Reply:
The brace notations only work in javadoc, not in comments.